packages/frontend/web/components/ArticleBody.tsx

```diff
@@ -529,27 +437,11 @@ export const ArticleBody: React.FC<{
                     />
                 </div>
                 <div className={cx(meta, guardianLines)}>
-                    <div className={cx(profile, pillarColours[CAPI.pillar])}>
-                        <span className={byline}>
-                            <RenderByline
-                                bylineText={CAPI.author.byline}
-                                contributorTags={CAPI.tags}
-                                pillar={CAPI.pillar}
-                            />
-                        </span>
-                    </div>
-                    {CAPI.author.twitterHandle && (
-                        <div className={twitterHandle}>
-                            <TwitterIcon />
-                            <a
-                                href={`https://www.twitter.com/${
-                                    CAPI.author.twitterHandle
-                                }`}
-                            >
-                                @{CAPI.author.twitterHandle}
-                            </a>
-                        </div>
-                    )}
+                    <Byline
+                        author={CAPI.author}
+                        tags={CAPI.tags}
+                        className={pillarColours[CAPI.pillar]}
```

Comment:
We're passing the pillar colour down as a class generated by Emotion. Is this the right approach? Or should we pass the hex colour down directly? I'm reluctant to pass the pillar down, as I'm not sure the GUUI component should know about pillars. Happy to get schooled otherwise

Reply:
I think they should. Pillars are essential to our article design and the alternative is to reach in quite deep inside the component to style it, and if we were to add another child, say, a button, suddenly everything that consumes this component has to also reach into the button to style it

Reply:
Okay, thanks for saying so, makes perfect sense. 

To preserve type safety, we would need to move `pillarMap` over to GUUI too, which would be painful to do as part of this PR. I also would like to discuss this with the team a bit too.

So either:

- leave it like this for now
- have non-type-safe pillar colours in `@guardian/guui/Byline` for now

I think I'm reasonably happy with the latter temporarily, but do shout if this concerns you.

Reply:
I'm happy with whatever makes you happy for now!